CHANGELOG.md

```diff
@@ -5,14 +5,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Released Versions
+
+* [1.1.0](#1.1.0)
+* [1.0.0](#1.0.0)
+
 ## [Upcoming]
 
 ### Track User Spending 
 
 - (IN PROGRESS) Working on preset event interfaces for tracking user spending. By 
   using these interfaces Integrators of MIKROS will be able to see information regarding 
   user spending habits e.g. how much and frequency of spending via Spending Score.
+  
+<a name="1.1.0"></a>
+## [1.1.0] - 2021-12-17
+
+### Platform Compatibility
+
+- Compatibile for both iOS and Android platforms.
+
+### Added Interfaces
+
+- TrackHandledException(): Use this to log any handled exceptions. This should be added to try-catch exceptions 
```

Comment:
A change needed here.
```
TrackHandledExceptionRequest.Builder()
    .SetException(exception)
    .Create(
    trackHandledExceptionRequest => MikrosManager.Instance.AnalyticsController.LogEvent(trackHandledExceptionRequest),
    onFailure =>
    {
        // handle failure
    });
```

Reply:
@Debadri-Redapple This is a CHANGELOG, not a README with examples for how to use the function. This example goes into the MIKROS SDK README, and also this repo README (which isn't updated yet).

Reply:
@Debadri-Redapple I did update to the CHANGELOG to represent that it is a request --> `TrackHandledExceptionRequest()`. That should be more clear.

Reply:
Yes, that's what I was implying.